### PR TITLE
[stable/zetcd] zetcd chart

### PR DESCRIPTION
**Pull request by hunter**
A chart that bootstraps zetcd with an optional etcd-operator cluster.

### CODE REVIEW

File: stable/zetcd/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+description: CoreOS zetcd Helm chart for Kubernetes
+name: zetcd
+version: 0.1.0
+appVersion: 0.0.3
+home: https://github.com/coreos/zetcd
+sources:
+  - https://github.com/coreos/zetcd
```
Comment (unguiculus) [inline stable/zetcd/Chart.yaml]: Please add maintainer. Use Github username.

File: stable/zetcd/values.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+service:
+  name: zetcd
+  type: ClusterIP
+  externalPort: 2181
+  internalPort: 2181
+resources:
+  # We usually recommend not to specify default resources and to leave this as a conscious 
+  # choice for the user. This also increases chances charts run on environments with little 
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following 
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
```
Comment (unguiculus) [inline stable/zetcd/values.yaml]: I'd rather not add defaults here.

**Review by unguiculus — CHANGES_REQUESTED** [review]
Thanks for the PR. Just a few things.

File: stable/zetcd/values.yaml (hunk 2 above)
Comment (hunter) [reply stable/zetcd/values.yaml]: Agreed. I'd forgotten to comment them back out after development. Now fixed.

File: stable/zetcd/Chart.yaml (hunk 1 above)
Comment (hunter) [reply stable/zetcd/Chart.yaml]: Added in maintainers

File: stable/zetcd/values.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+image:
+  repository: quay.io/coreos/zetcd
+  tag: v0.0.3
+  pullPolicy: IfNotPresent
+service:
+  name: zetcd
+  type: ClusterIP
+  externalPort: 2181
+  internalPort: 2181
+resources:
```
Comment (unguiculus) [inline stable/zetcd/values.yaml]: Add `{} ` again.

Comment (hunter) [reply stable/zetcd/values.yaml]: Apologies. Overlooked it :|

File: stable/zetcd/README.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+| ------------------------------------------------- | -------------------------------------------------------------------- | ---------------------------------------------- |
+| `replicaCount`                                    | Number of zetcd replicas to create     | `1`                                            |
+| `image.repository`                                | zetcd container image                                        | `quay.io/coreos/zetcd`                 |
+| `image.tag`                                       | zetcd container image tag                                    | `v0.0.3`                                       |
+| `image.pullPolicy`                                | zetcd container image pull policy                            | `IfNotPresent`                                 |
+| `resources.limits.cpu`                            | CPU limit per zetcd pod                                      | `100m`                                         |
+| `resources.limits.memory`                         | Memory limit per zetcd pod                                   | `128Mi`                                        |
```
Comment (unguiculus) [inline stable/zetcd/README.md]: Remove the defaults for resources.

File: stable/zetcd/templates/_helpers.tpl [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+{{- define "fullname" -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+{{- define "etcd-operator.fullname" -}}
```
Comment (unguiculus) [inline stable/zetcd/templates/_helpers.tpl]: This is not used. Why is it there?

**Review by unguiculus — CHANGES_REQUESTED** [review]
Almost there ;)

File: stable/zetcd/README.md (hunk 4 above)
Comment (hunter) [reply stable/zetcd/README.md]: Done :)

File: stable/zetcd/templates/_helpers.tpl (hunk 5 above)
Comment (hunter) [reply stable/zetcd/templates/_helpers.tpl]: Removed as its no longer being used

**Review by unguiculus — APPROVED** [review]

**Pull request merged**